README.md

```diff
@@ -13,6 +13,12 @@
 
 Streaming services for large datasets of Facility, Account, Compliance, & Emissions data.
 
+## Important Notice
+
+**This service is configured to use a read replica database connection by default.** All database traffic is routed to the read replica whenever possible to optimize performance for data streaming operations.
+
+If database-modifying endpoints are added to this service in the future, developers must ensure that those operations explicitly use the primary database connection instead of the replica. The current configuration automatically routes all queries to the read replica unless specifically overridden.
```

Comment:
When the default mode is "slave", CREATE, UPDATE, DELETE operations will still use the primary database, TypeORM takes care of that. The caveat is that if a SELECT operation relies on that updated data, that operation should manually be pointed to the primary server, otherwise it might retrieve stale data. It may be helpful to update this note to that effect.